### PR TITLE
Fredrik

### PR DESCRIPTION
**Pull request by fredrik-allenmark**
Please review my changes

### CODE REVIEW

File: discussion.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+
+## More discussion
+
+Lots of extra discussion here.
```
Comment (assumpcao01) [inline discussion.md]: You bet your ass!

File: discussion.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+## More discussion
```
Comment (strongway) [inline discussion.md]: give one or two more examples (or citations)

File: discussion.md (hunk 1 above)
Comment (lingyue27) [inline discussion.md]: ↖(^ω^)↗

**Comment by iceliboying** [comment]
hello haha

File: discussion.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+
```
Comment (strongway) [inline discussion.md]: change facilitate to other words.

File: discussion.md (hunk 1 above)
Comment (janosmann) [inline discussion.md]: yeah, the need to be done some change

**Pull request merged**